tools/logrotate_config/package_after_install_script.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Create the log file for gcsfuse package.
+mkdir -p /var/log/gcsfuse/
```

Comment:
Ensure this directory is created with non root user. Also check for path starting with /etc/ below.

Reply:
Yes, done.